components/security-mgt/org.wso2.carbon.security.mgt/src/main/java/org/wso2/carbon/security/keystore/dao/impl/KeyStoreDAOImpl.java

```diff
@@ -0,0 +1,249 @@
+/*
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.security.keystore.dao.impl;
+
+import org.wso2.carbon.database.utils.jdbc.NamedPreparedStatement;
+import org.wso2.carbon.identity.core.util.IdentityDatabaseUtil;
+import org.wso2.carbon.security.keystore.KeyStoreManagementException;
+import org.wso2.carbon.security.keystore.dao.KeyStoreDAO;
+import org.wso2.carbon.security.keystore.dao.constants.KeyStoreDAOConstants;
+import org.wso2.carbon.security.keystore.dao.constants.KeyStoreDAOConstants.KeyStoreTableColumns;
+import org.wso2.carbon.security.keystore.model.KeyStoreModel;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Timestamp;
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.List;
+import java.util.Optional;
+import java.util.TimeZone;
+import java.util.UUID;
+
+import static java.time.ZoneOffset.UTC;
+
+/**
+ * This class provides the implementation of the KeyStoreDAO interface.
+ */
+public class KeyStoreDAOImpl implements KeyStoreDAO {
+
+    private final Calendar CALENDAR = Calendar.getInstance(TimeZone.getTimeZone(UTC));
+    private static final String DB_CONN_RETRIEVAL_ERROR_MSG = "Error while getting the DB connection.";
+
+    public KeyStoreDAOImpl() {
```

Comment:
We can remove the default constructor

Reply:
Resolved with 062799393567b744e3f873afffe2c87a67b07629